.circleci/config.yml

```diff
@@ -264,8 +261,7 @@ jobs:
     steps:
       - release_to_namespace:
           namespace: "prod"
-          establishment: "cookhamwood"
-          establishmentDisplayName: "HMYOI Cookham Wood"
+          establishment: "cw"
           releaseName: "prisoner-content-hub-frontend-cw"
```

Comment:
These two lines could do with updating to `cookhamwood` for consistency, and we should rename `values.cw.yaml` to `values.cookhamwood.yaml`

Reply:
Updated Circle and filename